modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CppPistacheServerCodegen.java

```diff
@@ -82,7 +90,8 @@ public CppPistacheServerCodegen() {
         addSwitch(OPTIONAL_EXTERNAL_LIB, OPTIONAL_EXTERNAL_LIB_DESC, this.isAddExternalLibs);
         addOption(HELPERS_PACKAGE_NAME, HELPERS_PACKAGE_NAME_DESC, this.helpersPackage);
 
-        reservedWords = new HashSet<>();
+        // TODO @Fjolnir-Dvorak
+//        reservedWords = new HashSet<>();
```

Comment:
is this still needed?

Reply:
I do not understand the intention of setting reservedWords to an empty HashSet overwriting the parent behavior so I added that just to be save to not accidentally destroy that generator.

Reply:
Resolved in commit [11ff0928](https://github.com/OpenAPITools/openapi-generator/commit/11ff0928fd16a8a9342c43b65026e1c4dc973d3d)